### PR TITLE
NPS(feature): add nps collector

### PR DESCRIPTION
**Pull request by rebortg**
The PR add the a NPS collector of server data from this 2 classes:

- Win32_PerfRawData_IAS_NPSAuthenticationServer 
- Win32_PerfRawData_IAS_NPSAccountingServer

### CODE REVIEW

File: collector/nps.go [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+
+import (
+	"github.com/prometheus-community/windows_exporter/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/yusufpapurcu/wmi"
+)
+
+func init() {
+	registerCollector("nps", newnpsCollector) // TODO: Add any perflib dependencies here
+}
```
Comment (breed808) [inline collector/nps.go:11]: The `init` function here will need to be removed, with the collector being registered in the [init.go](https://github.com/prometheus-community/windows_exporter/blob/b033ff71ffbd85e9774a9bdcc25c9cd91d1e5e8e/collector/init.go#L33) file.

File: collector/nps.go [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+	AccountingMalformedPackets  *prometheus.Desc
+	AccountingNoRecord          *prometheus.Desc
+	AccountingPacketsReceived   *prometheus.Desc
+	AccountingPacketsSent       *prometheus.Desc
+	AccountingServerResetTime   *prometheus.Desc
+	AccountingServerUpTime      *prometheus.Desc
+	AccountingUnknownType       *prometheus.Desc
+}
+
+func newnpsCollector() (Collector, error) {
```
Comment (breed808) [inline collector/nps.go:44]: Function should be renamed to `newNPSCollector` to fix the failing CI job.

File: collector/nps.go [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+	}
+	if desc, err := c.CollectAccounting(ch); err != nil {
+		log.Error("failed collecting NPS accounting data:", desc, err)
+		return err
+	}
+	return nil
+}
+
+// Win32_PerfRawData_IAS_NPSAuthenticationServer docs:
+// - <add link to documentation here>
```
Comment (breed808) [inline collector/nps.go:216]: Is there a documentation link for the two classes?

**Review by breed808 — CHANGES_REQUESTED** [review]
Thanks for submitting this! Let me know if any clarification is needed on the comments.

**Review by breed808 — CHANGES_REQUESTED** [review]
Removal of the `init()` function is good, but you need to add an entry in the `collectors` slice in [collectors/init.go](https://github.com/prometheus-community/windows_exporter/blob/master/collector/init.go#L28)

**Comment by rebortg** [comment]
Done. I hope now all is correct

**Review by breed808 — APPROVED** [review]

**Pull request merged**